portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyTestFile.testcase

```diff
@@ -0,0 +1,22 @@
+@component-name = "portal"
+definition {
+
+	property testray.main.component.name = "Navigation";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+	}
+
+	@description = "Access Portal Global Menu and validate CONFIGURATION string at Control Panel"
+	@priority = 5
+	test AssertConfigurationSectionPresentInApplicationsMenu {
+		MyMacro.gotoControlPanel();
+
+		AssertElementPresent(locator1 = "MyPath#PANEL_CONFIGURATION_SECTION");
+	}
```

Comment:
Hey, I know that this line is pretty simple and we probably don`t need a macro for this, but since the aim of the exercise is for you to do a macro, can you add a macro just to assert the configuration text is present?
The macro will be simple, you just have to put a function of assertTextEquals inside the macro and pass the value (the string) to the macro.

or if you want to level up the things a little bit, I believe you can try using a new XPath like this one:
`//h2[@id='nav_control_panel.${yourVariable}']`
doing that you would be able to validate all the strings related to the title, if you take a look at the HTML code you will see that all the ids are very similar except for the last part:

- nav_control_panel.configuration
- nav_control_panel.notifications
- nav_control_panel.security

The last part will be your variable, so when creating a macro you could use the assertTextEquals passing the variable of the XPath and the value you are expecting.

Reply:
That was actually really good! Thanks for the tip and the review, I'll keep that in mind to improve my Poshi skills.